ratatool-sampling/README.md

```diff
@@ -24,6 +24,7 @@ Usage: ratatool bigSampler [dataflow_options] [options]
   --output=<path>                            Output file path or BigQuery table
   [--fields=<field1,field2,...>]             An optional list of fields to include in hashing for sampling cohort selection
   [--seed=<seed>]                            An optional seed used in hashing for sampling cohort selection
+  [--hashAlgorithm=(murmur|farm)]            An optional arg to select the hashing algorithm used in conjunction with the seed. Either MurmurHash or FarmHash. Default is FarmHash.
```

Comment:
We should match the text of the prior fields.  Something along the lines of:
`An optional are to select the hashing algorithm for sampling cohort selection. Defaults to FarmHash for BigQuery compatibility`

Can save some space by not specifying farmhash and murmurhash since they are in the arg options to the left.